Source/Wikitivity.Agent/WikitivityAgentHelper.cs

```diff
@@ -101,28 +214,41 @@ public List<WikitivityUploadsAgent.DataObtainedSingleRequestObject> PopulateArti
 
 		public static string GetInstanceUrl(IAgentHelper helper, AgentBase aB)
 		{
-			string instanceUrl = "";
 			try
 			{
 				aB.RaiseMessage("Attempting to get URL...", 1);
-				instanceUrl = helper.GetInstanceSettingBundle().GetString("Relativity.Core", "KeplerServicesUri");
+				//instanceUrl = helper.GetInstanceSettingBundle().GetString("Relativity.Core", "KeplerServicesUri");
+				string instanceUrl = helper.GetInstanceSettingBundle().GetStringAsync("Relativity.Wikitivity", "WikitivityWebApi").Result;

```

Comment:
I would store all the hard-coded strings in a Constants class so they can be easily identified in one place and easily updated in future.